api/app/signals/apps/signals/managers.py

```diff
@@ -58,6 +60,13 @@ def _create_initial_no_transaction(self, signal_data, location_data, status_data
         default_stadsdeel = location_data['stadsdeel'] if 'stadsdeel' in location_data else None
         location_data['stadsdeel'] = _get_stadsdeel_code(location_data['geometrie'], default_stadsdeel)
 
+        # set area_type and area_code if default area type is provided
+        if DEFAULT_SIGNAL_AREA_TYPE:
```

Comment:
We should have a test that checks that these properties are correctly set on signal creation.